### ISSUE TITLE
2025-05-05, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#160]
# Today's Arxiv Papers

(1) [TRAVELER: A Benchmark for Evaluating Temporal Reasoning across Vague, Implicit and Explicit References](http://arxiv.org/abs/2505.01325v1), 2025-05-02

(2) [Helping Big Language Models Protect Themselves: An Enhanced Filtering and Summarization System](http://arxiv.org/abs/2505.01315v1), 2025-05-02

(3) [Enhancing SPARQL Query Rewriting for Complex Ontology Alignments](http://arxiv.org/abs/2505.01309v1), 2025-05-02

(4) [Document Retrieval Augmented Fine-Tuning (DRAFT) for safety-critical software assessments](http://arxiv.org/abs/2505.01307v1), 2025-05-02

(5) [FlowDubber: Movie Dubbing with LLM-based Semantic-aware Learning and Flow Matching based Voice Enhancing](http://arxiv.org/abs/2505.01263v1), 2025-05-02

(6) [Digital Pathway Curation (DPC): a comparative pipeline to assess the reproducibility, consensus and accuracy across Gemini, PubMed, and scientific reviewers in biomedical research](http://arxiv.org/abs/2505.01259v1), 2025-05-02

(7) [EvalxNLP: A Framework for Benchmarking Post-Hoc Explainability Methods on NLP Models](http://arxiv.org/abs/2505.01238v1), 2025-05-02

(8) [CaReAQA: A Cardiac and Respiratory Audio Question Answering Model for Open-Ended Diagnostic Reasoning](http://arxiv.org/abs/2505.01199v1), 2025-05-02

(9) [LLM Security: Vulnerabilities, Attacks, Defenses, and Countermeasures](http://arxiv.org/abs/2505.01177v1), 2025-05-02

(10) [On the Limitations of Steering in Language Model Alignment](http://arxiv.org/abs/2505.01162v1), 2025-05-02

(11) [Methodological Foundations for AI-Driven Survey Question Generation](http://arxiv.org/abs/2505.01150v1), 2025-05-02

(12) [Retrieval-Augmented Generation in Biomedicine: A Survey of Technologies, Datasets, and Clinical Applications](http://arxiv.org/abs/2505.01146v1), 2025-05-02

(13) [MateICL: Mitigating Attention Dispersion in Large-Scale In-Context Learning](http://arxiv.org/abs/2505.01110v1), 2025-05-02

(14) [MADIL: An MDL-based Framework for Efficient Program Synthesis in the ARC Benchmark](http://arxiv.org/abs/2505.01081v1), 2025-05-02

(15) [Zero-Shot Document-Level Biomedical Relation Extraction via Scenario-based Prompt Design in Two-Stage with LLM](http://arxiv.org/abs/2505.01077v1), 2025-05-02

(16) [WirelessAgent: Large Language Model Agents for Intelligent Wireless Networks](http://arxiv.org/abs/2505.01074v1), 2025-05-02

(17) [Retrieval Augmented Learning: A Retrial-based Large Language Model Self-Supervised Learning and Autonomous Knowledge Generation](http://arxiv.org/abs/2505.01073v1), 2025-05-02

(18) [A Rusty Link in the AI Supply Chain: Detecting Evil Configurations in Model Repositories](http://arxiv.org/abs/2505.01067v1), 2025-05-02

(19) [Good News for Script Kiddies? Evaluating Large Language Models for Automated Exploit Generation](http://arxiv.org/abs/2505.01065v1), 2025-05-02

(20) [Efficient Vocabulary-Free Fine-Grained Visual Recognition in the Age of Multimodal LLMs](http://arxiv.org/abs/2505.01064v1), 2025-05-02

(21) [Transferable Adversarial Attacks on Black-Box Vision-Language Models](http://arxiv.org/abs/2505.01050v1), 2025-05-02

(22) [Low-Precision Training of Large Language Models: Methods, Challenges, and Opportunities](http://arxiv.org/abs/2505.01043v1), 2025-05-02

(23) [Do We Need a Detailed Rubric for Automated Essay Scoring using Large Language Models?](http://arxiv.org/abs/2505.01035v1), 2025-05-02

(24) [Value Portrait: Understanding Values of LLMs with Human-aligned Benchmark](http://arxiv.org/abs/2505.01015v1), 2025-05-02

(25) [Improving Large Language Model Planning with Action Sequence Similarity](http://arxiv.org/abs/2505.01009v1), 2025-05-02

(26) [Togedule: Scheduling Meetings with Large Language Models and Adaptive Representations of Group Availability](http://arxiv.org/abs/2505.01000v1), 2025-05-02

(27) [VTS-LLM: Domain-Adaptive LLM Agent for Enhancing Awareness in Vessel Traffic Services through Natural Language](http://arxiv.org/abs/2505.00989v1), 2025-05-02

(28) [Position: Enough of Scaling LLMs! Lets Focus on Downscaling](http://arxiv.org/abs/2505.00985v1), 2025-05-02

(29) [Multi-agents based User Values Mining for Recommendation](http://arxiv.org/abs/2505.00981v1), 2025-05-02

(30) [Synthesize-on-Graph: Knowledgeable Synthetic Data Generation for Continue Pre-training of Large Language Models](http://arxiv.org/abs/2505.00979v1), 2025-05-02

(31) [Attack and defense techniques in large language models: A survey and new perspectives](http://arxiv.org/abs/2505.00976v1), 2025-05-02

(32) [Generating Animated Layouts as Structured Text Representations](http://arxiv.org/abs/2505.00975v1), 2025-05-02

(33) [Seeking to Collide: Online Safety-Critical Scenario Generation for Autonomous Driving with Retrieval Augmented Large Language Models](http://arxiv.org/abs/2505.00972v1), 2025-05-02

(34) [Preserving Privacy and Utility in LLM-Based Product Recommendations](http://arxiv.org/abs/2505.00951v1), 2025-05-02

(35) [SSRLBot: Designing and Developing an LLM-based Agent using Socially Shared Regulated Learning](http://arxiv.org/abs/2505.00945v1), 2025-05-02

(36) [Large Language Model-Driven Dynamic Assessment of Grammatical Accuracy in English Language Learner Writing](http://arxiv.org/abs/2505.00931v1), 2025-05-02

(37) [How Transformers Learn Regular Language Recognition: A Theoretical Study on Training Dynamics and Implicit Bias](http://arxiv.org/abs/2505.00926v1), 2025-05-02

(38) [Multivariate Conformal Selection](http://arxiv.org/abs/2505.00917v1), 2025-05-01

(39) [NeMo-Inspector: A Visualization Tool for LLM Generation Analysis](http://arxiv.org/abs/2505.00903v1), 2025-05-01

(40) [Towards Explainable Temporal User Profiling with LLMs](http://arxiv.org/abs/2505.00886v1), 2025-05-01

(41) [Protocol-agnostic and Data-free Backdoor Attacks on Pre-trained Models in RF Fingerprinting](http://arxiv.org/abs/2505.00881v1), 2025-05-01

(42) [Thoughts without Thinking: Reconsidering the Explanatory Value of Chain-of-Thought Reasoning in LLMs through Agentic Pipelines](http://arxiv.org/abs/2505.00875v1), 2025-05-01

(43) [LLM Ethics Benchmark: A Three-Dimensional Assessment System for Evaluating Moral Reasoning in Large Language Models](http://arxiv.org/abs/2505.00853v1), 2025-05-01

(44) [ICQuant: Index Coding enables Low-bit LLM Quantization](http://arxiv.org/abs/2505.00850v1), 2025-05-01

(45) [OET: Optimization-based prompt injection Evaluation Toolkit](http://arxiv.org/abs/2505.00843v1), 2025-05-01

(46) [From Texts to Shields: Convergence of Large Language Models and Cybersecurity](http://arxiv.org/abs/2505.00841v1), 2025-05-01

(47) [SmallPlan: Leverage Small Language Models for Sequential Path Planning with Simulation-Powered, LLM-Guided Distillation](http://arxiv.org/abs/2505.00831v1), 2025-05-01

(48) [Should AI Mimic People? Understanding AI-Supported Writing Technology Among Black Users](http://arxiv.org/abs/2505.00821v1), 2025-05-01

(49) [HMCF: A Human-in-the-loop Multi-Robot Collaboration Framework Based on Large Language Models](http://arxiv.org/abs/2505.00820v1), 2025-05-01

(50) [Spill The Beans: Exploiting CPU Cache Side-Channels to Leak Tokens from Large Language Models](http://arxiv.org/abs/2505.00817v1), 2025-05-01

(51) [Reasoning Capabilities and Invariability of Large Language Models](http://arxiv.org/abs/2505.00776v1), 2025-05-01

(52) [T2I-R1: Reinforcing Image Generation with Collaborative Semantic-level and Token-level CoT](http://arxiv.org/abs/2505.00703v1), 2025-05-01

(53) [Multi-Modal Language Models as Text-to-Image Model Evaluators](http://arxiv.org/abs/2505.00759v1), 2025-05-01

(54) [MINERVA: Evaluating Complex Video Reasoning](http://arxiv.org/abs/2505.00681v1), 2025-05-01

(55) [Steering Large Language Models with Register Analysis for Arbitrary Style Transfer](http://arxiv.org/abs/2505.00679v1), 2025-05-01

(56) [Rethinking Memory in AI: Taxonomy, Operations, Topics, and Future Directions](http://arxiv.org/abs/2505.00675v1), 2025-05-01

(57) [DeepCritic: Deliberate Critique with Large Language Models](http://arxiv.org/abs/2505.00662v1), 2025-05-01

(58) [On the generalization of language models from in-context learning and finetuning: a controlled study](http://arxiv.org/abs/2505.00661v1), 2025-05-01

(59) [Large Language Models Understanding: an Inherent Ambiguity Barrier](http://arxiv.org/abs/2505.00654v1), 2025-05-01

(60) [Open-Source LLM-Driven Federated Transformer for Predictive IoV Management](http://arxiv.org/abs/2505.00651v1), 2025-05-01

(61) [Investigating Task Arithmetic for Zero-Shot Information Retrieval](http://dx.doi.org/10.1145/3726302.3730216), 2025-05-01

(62) [The Illusion of Role Separation: Hidden Shortcuts in LLM Role Learning (and How to Fix Them)](http://arxiv.org/abs/2505.00626v1), 2025-05-01

(63) [FineScope : Precision Pruning for Domain-Specialized Large Language Models Using SAE-Guided Self-Data Cultivation](http://arxiv.org/abs/2505.00624v1), 2025-05-01

(64) [Combining LLMs with Logic-Based Framework to Explain MCTS](http://arxiv.org/abs/2505.00610v1), 2025-05-01

(65) [Can LLMs Help Improve Analogical Reasoning For Strategic Decisions? Experimental Evidence from Humans and GPT-4](http://arxiv.org/abs/2505.00603v1), 2025-05-01

(66) [Block Circulant Adapter for Large Language Models](http://arxiv.org/abs/2505.00582v1), 2025-05-01

(67) [FreqKV: Frequency Domain Key-Value Compression for Efficient Context Window Extension](http://arxiv.org/abs/2505.00570v1), 2025-05-01

(68) [Triggering Hallucinations in LLMs: A Quantitative Study of Prompt-Induced Hallucination in Large Language Models](http://arxiv.org/abs/2505.00557v1), 2025-05-01

(69) [100 Days After DeepSeek-R1: A Survey on Replication Studies and More Directions for Reasoning Language Models](http://arxiv.org/abs/2505.00551v2), 2025-05-01

(70) [Leveraging Partial SMILES Validation Scheme for Enhanced Drug Design in Reinforcement Learning Frameworks](http://arxiv.org/abs/2505.00530v1), 2025-05-01

(71) [HalluMix: A Task-Agnostic, Multi-Domain Benchmark for Real-World Hallucination Detection](http://arxiv.org/abs/2505.00506v1), 2025-05-01

(72) [UserCentrix: An Agentic Memory-augmented AI Framework for Smart Spaces](http://arxiv.org/abs/2505.00472v1), 2025-05-01

(73) [Red Teaming Large Language Models for Healthcare](http://arxiv.org/abs/2505.00467v1), 2025-05-01

(74) [Data Therapist: Eliciting Domain Knowledge from Subject Matter Experts Using Large Language Models](http://arxiv.org/abs/2505.00455v1), 2025-05-01

(75) [Distributed Retrieval-Augmented Generation](http://arxiv.org/abs/2505.00443v1), 2025-05-01

(76) [A Survey on Large Language Model based Human-Agent Systems](http://arxiv.org/abs/2505.00753v1), 2025-05-01

(77) [Urban Air Mobility as a System of Systems: An LLM-Enhanced Holonic Approach](http://arxiv.org/abs/2505.00368v1), 2025-05-01

(78) [KoACD: The First Korean Adolescent Dataset for Cognitive Distortion Analysis](http://arxiv.org/abs/2505.00367v1), 2025-05-01

(79) [LLMPrism: Black-box Performance Diagnosis for Production LLM Training Platforms](http://arxiv.org/abs/2505.00342v1), 2025-05-01

(80) [Communication-Efficient Wireless Federated Fine-Tuning for Large-Scale AI Models](http://arxiv.org/abs/2505.00333v1), 2025-05-01

(81) [Mixture of Sparse Attention: Content-Based Learnable Sparse Attention via Expert-Choice Routing](http://arxiv.org/abs/2505.00315v1), 2025-05-01

(82) [Gateformer: Advancing Multivariate Time Series Forecasting through Temporal and Variate-Wise Attention with Gated Representations](http://arxiv.org/abs/2505.00307v1), 2025-05-01

(83) [AdCare-VLM: Leveraging Large Vision Language Model (LVLM) to Monitor Long-Term Medication Adherence and Care](http://arxiv.org/abs/2505.00275v1), 2025-05-01

(84) [Large Language Models as AI Agents for Digital Atoms and Molecules: Catalyzing a New Era in Computational Biophysics](http://arxiv.org/abs/2505.00270v1), 2025-05-01

(85) [InstructAttribute: Fine-grained Object Attributes editing with Instruction](http://arxiv.org/abs/2505.00751v1), 2025-05-01

(86) [EnronQA: Towards Personalized RAG over Private Documents](http://arxiv.org/abs/2505.00263v1), 2025-05-01

(87) [LLM-Based Threat Detection and Prevention Framework for IoT Ecosystems](http://arxiv.org/abs/2505.00240v1), 2025-05-01

(88) [Self-Generated In-Context Examples Improve LLM Agents for Sequential Decision-Making Tasks](http://arxiv.org/abs/2505.00234v2), 2025-05-01

(89) [Which Agent Causes Task Failures and When? On Automated Failure Attribution of LLM Multi-Agent Systems](http://arxiv.org/abs/2505.00212v1), 2025-04-30

(90) [RAIL in the Wild: Operationalizing Responsible AI Evaluation Using Anthropic's Value Dataset](http://arxiv.org/abs/2505.00204v1), 2025-04-30

(91) [V3LMA: Visual 3D-enhanced Language Model for Autonomous Driving](http://arxiv.org/abs/2505.00156v1), 2025-04-30

(92) [Audo-Sight: Enabling Ambient Interaction For Blind And Visually Impaired Individuals](http://arxiv.org/abs/2505.00153v1), 2025-04-30

(93) [AdaptMI: Adaptive Skill-based In-context Math Instruction for Small Language Models](http://arxiv.org/abs/2505.00147v1), 2025-04-30

(94) [When Deep Learning Meets Information Retrieval-based Bug Localization: A Survey](http://arxiv.org/abs/2505.00144v1), 2025-04-30

(95) [Between Underthinking and Overthinking: An Empirical Study of Reasoning Length and correctness in LLMs](http://arxiv.org/abs/2505.00127v1), 2025-04-30

(96) [Fine-Tuning LLMs for Low-Resource Dialect Translation: The Case of Lebanese](http://arxiv.org/abs/2505.00114v1), 2025-04-30

(97) [CoordField: Coordination Field for Agentic UAV Task Allocation In Low-altitude Urban Scenarios](http://arxiv.org/abs/2505.00091v1), 2025-04-30

(98) [TRUST: An LLM-Based Dialogue System for Trauma Understanding and Structured Assessments](http://arxiv.org/abs/2504.21851v1), 2025-04-30

(99) [COMPACT: COMPositional Atomic-to-Complex Visual Capability Tuning](http://arxiv.org/abs/2504.21850v1), 2025-04-30

(100) [An Empirical Study on the Effectiveness of Large Language Models for Binary Code Understanding](http://arxiv.org/abs/2504.21803v1), 2025-04-30
